lib/src/main/java/com/auth0/android/lock/views/SocialButton.java

```diff
@@ -69,24 +80,19 @@ private StateListDrawable getTouchFeedbackBackground(@ColorInt int pressedColor,
     public void setStyle(AuthConfig config, @AuthMode int mode) {
         final Drawable logo = config.getLogo(getContext());
         final int backgroundColor = config.getBackgroundColor(getContext());
-        Drawable touchBackground = getTouchFeedbackBackground(backgroundColor, ViewUtils.Corners.ONLY_RIGHT);
+        Drawable touchBackground = getTouchFeedbackBackground(backgroundColor, ViewUtils.Corners.ALL);
```

Comment:
all corners since now it's being applied to the parent view (The whole clickable button)